src/dso_api/dynamic_api/serializers.py

```diff
@@ -154,6 +154,8 @@ def serializer_factory(
 ) -> Type[DynamicSerializer]:
     """Generate the DRF serializer class for a specific dataset model."""
     fields = ["_links", "schema"]
+    if isinstance(model, str):
+        raise Exception(f"Model {model} could not be resolved.")
```

Comment:
Misschien het is beter om `django.core.exceptions. ImproperlyConfigured` hier te gebruiken om fout iets duidelijker te makken?
 https://docs.djangoproject.com/en/3.1/ref/exceptions/#improperlyconfigured

Reply:
Goeie suggestie, ga ik erin zetten.